src/Common/src/Interop/Unix/System.Native/Interop.Access.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        internal enum AccessMode : int
+        {
+            F_OK = 0,   /* Check for existence */
+            X_OK = 1,   /* Check for execute */
+            W_OK = 2,   /* Check for write */
+            R_OK = 4,   /* Check for read */
+        }
+
+        [DllImport(Libraries.SystemNative, SetLastError = true)]
+        internal static extern int Access(
+            [MarshalAs(UnmanagedType.LPStr)]
+            string path, 
```

Comment:
I'd prefer in cases like this for the attribute to be on the same line as the parameter. (That said, in this case the attribute shouldn't even be necessary, as it should be the default.  Another alternative if you did want to be explicit would be to use CharSet in the DllImport.)